lib/BackgroundJob/Reminder.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * @copyright Copyright (c) 2023 Joas Schilling <coding@schilljs.com>
+ *
+ * @author Joas Schilling <coding@schilljs.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Talk\BackgroundJob;
+
+use OCA\Talk\Service\ReminderService;
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\TimedJob;
+
+class Reminder extends TimedJob {
+	public function __construct(
+		ITimeFactory $time,
+		protected ReminderService $reminderService,
+	) {
+		parent::__construct($time);
+		// Every time (5 minutes minus 1 second)
```

Comment:
Why "minus 1 second"?

I also wonder if we can set the interval to match the next reminder. It would also mean to change the interval if a new reminder is set before… 
It's not in the scope of this pull request of course :) Just a thought

Reply:
> Why "minus 1 second"?

I was thinking about setting it to 1 sec in the beginning. It's weird edgecasing in my head I guess.

If the cron job runs at 8:55AM and the next time 5min later at 9:00AM it would be skipped there, because it's not 300 seconds since the last run. But on our instance that would not be enough as an "empty" run without heavy jobs takes like 5 seconds. So maybe we just set it to a minute.